website/archiver/model.py

```diff
@@ -0,0 +1,154 @@
+import datetime
+
+from modularodm import (
+    fields,
+    Q,
+)
+
+from framework.mongo import ObjectId
+from framework.mongo import StoredObject
+
+from website.archiver import (
+    ARCHIVER_INITIATED,
+    ARCHIVER_SUCCESS,
+    ARCHIVER_FAILURE,
+    ARCHIVER_FAILURE_STATUSES
+)
+
+from website.addons.base import StorageAddonBase
+from website import settings
+
+class ArchiveTarget(StoredObject):
+
+    _id = fields.StringField(
+        primary=True,
+        default=lambda: str(ObjectId())
+    )
+
+    name = fields.StringField()
+
+    status = fields.StringField(default=ARCHIVER_INITIATED)
+    stat_result = fields.DictionaryField()
+    errors = fields.StringField(list=True)
+
+    def __repr__(self):
+        return "{0}: {1}".format(self.name, self.status)
```

Comment:
For clarity sake you may want to wrap this with `<classname(...)>` or something like that

This is the generic repr we use in waterbutler

```python
    def __repr__(self):
        return '{}(..)'.format(self.__class__.__name__, ...)  # Or add <> around as well

```